src/oscar_accounts/abstract_models.py

```diff
@@ -237,11 +237,18 @@ def days_remaining(self, from_date=None):
         return (self.end_date - from_date).days
 
     def close(self):
-        # Only account with zero balance can be closed
+        """This object must be in the database before calling this function.
+
+        Only account with zero balance can be closed
+
+        Saves these changes to the database but doesn't save any other changes you may have made to the object.
+
+        Changed in version 2.2
```

Comment:
Could you please remove the "Changed in version 2.2" from the dosctrings? It's the sort of thing that ends up lying around long after the change is no longer a recent concern. We'll add it to the changelog for the next release. Also think we can probably omit the "This object must be in the database before calling this function" as it's quite unlikely anyone is going to run into that outside of a unit test.

Reply:
Absolutely, I can make the changes 😊 I'll get the changes made tomorrow morning!

The "Changed in X.X" was inspired by [discord.py](https://github.com/Rapptz/discord.py) as I saw they used it extensively in their project and thought it would be a good addition. Mind you, I also struggled to find the part of your code which defines what version number we're actually on 😅

Reply:
I totally forgot! Fixed!